cylc/flow/data_store_mgr.py

```diff
@@ -330,6 +341,9 @@ def apply_delta(key, delta, data):
                         'on update application: %s' % str(exc)
                     )
                     continue
+    # Clear memory accumulation
+    if key == WORKFLOW:
+        data[key] = reset_protobuf_object(MESSAGE_MAP[key], data[key])
```

Comment:
`PbWorkflow` is the only long-lived protobuf object in the store, all the other nodes/edges/objects are transitory (added then eventually pruned)..

Reply:
What is it about PbWorkflow that makes it long-lived by contrast to the other fields?

Reply:
Because we never delete it, we only add-to/update/prune-from fields.